src/NServiceBus.Core/BusConfiguration.cs

```diff
@@ -61,36 +60,71 @@ public void RegisterComponents(Action<IConfigureComponents> registration)
         }
 
         /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="assemblies"></param>
+        public void ExcludeAssemblies(params string[] assemblies)
+        {
+            if (assemblies == null)
+            {
+                throw new ArgumentNullException("assemblies");
+            }
+
+            if (assemblies.Any(string.IsNullOrWhiteSpace))
+            {
+                throw new ArgumentException("Passed in a null or empty assembly name.", "assemblies");
+            }
+            excludedAssemblies = excludedAssemblies.Union(assemblies, StringComparer.OrdinalIgnoreCase).ToList();
```

Comment:
Cool so this makes it additive. Nice. Therefore we can now tackle the issues with the testing framework and xunit exclusions, right?